api/v1beta1/verticareplicator_types.go

```diff
@@ -22,12 +22,62 @@ import (
 
 // VerticaReplicatorSpec defines the desired state of VerticaReplicator
 type VerticaReplicatorSpec struct {
+	// +kubebuilder:validation:Required
+	// +operator-sdk:csv:customresourcedefinitions:type=spec
+	// +operator-sdk:csv:customresourcedefinitions:type=spec,xDescriptors="urn:alm:descriptor:com.tectonic.ui:advanced"
```

Comment:
```suggestion
```
This should not have the `advanced` marker as it is a required field.

Reply:
should I just remove `xDescriptors="urn:alm:descriptor:com.tectonic.ui:advanced"`?